Include/compile.h

```diff
@@ -24,6 +24,8 @@ PyAPI_FUNC(PyCodeObject *) PyNode_Compile(struct _node *, const char *);
 #define PyCF_IGNORE_COOKIE 0x0800
 #define PyCF_TYPE_COMMENTS 0x1000
 #define PyCF_ALLOW_TOP_LEVEL_AWAIT 0x2000
```

Comment:
How many bits are free for incoming compiler flags? Two? 0x4000 and 0x8000?

Reply:
and `0x10000` (now that we removed that old macro)